### PR TITLE
Fix English grammar

### PR DESCRIPTION
**Pull request by lu-na-r**
A pull request to fix some grammar mistakes in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ For more details, please see README for each action.
 
 ## GUI version of vrc-get
 
-Since later February 2024, vrc-get has experimental gui version.
+Since later February 2024, vrc-get has been experimenting a gui version.
```
Comment (anatawa12) [inline README.md:119]: 実験してるのではなくexperimentalなgui版がありますとしたいです。experimentalが用語に近い認識があります。

**Review by anatawa12 — COMMENTED** [review]
ありがとうございます。一点確認をお願いします

File: README.md (hunk 1 above)
Comment (lu-na-r) [reply README.md]: Thank you for your reviewing.

I have changed the sentence to "vrc-get has had an experimental gui version". However, I am not sure if this is the best phrase. I will open another pull request if I hit on with a better idea.

Comment (anatawa12) [reply README.md]: 今もずっと継続しているので現在完了形も違和感がありますね。

Comment (lu-na-r) [reply README.md]: Just fixed the sentence to "an experimental gui version of vrc-get has been available".

This is still using the past perfect tense but much better and easier to understand the meaning. If you do not like this, you can say "vrc-get has been providing an experimental gui version" instead.

Comment (anatawa12) [reply README.md]: it looks good for me. thank you

**Pull request merged**